lib/models/builder.js

```diff
@@ -130,8 +127,10 @@ module.exports = CoreObject.extend({
 
   init: function() {
     this._super.apply(this, arguments);
+    this._buildCount = 0;
     this.setupBroccoliBuilder();
     this.trapSignals();
+    this._instantiationStack = (new Error).stack.replace(/[^\n]*\n/, '');
```

Comment:
I just found this helpful when I was tracking down the signal handlers that weren't being cleaned up properly (in tests mostly), I can remove it if others don't find it valuable..